README.md

```diff
@@ -1,21 +1,107 @@
-# Website
+# Get Started
 
 This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 
-## Installation
 
-```console
-yarn install
+
+## Dependencies
+
+* **Nodejs** version >= 14 or above
+
+* **Yarn** version >= 1.5
+
+* **NPM** [Guide](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+* **git**
+
+
+# How to Install
+
+## Install Dependencies for Windows users
+
+Download the [Nodejs installer](https://nodejs.org/en/download/) from their Official website.
+
+### Install Yarn
+
+```
+npm install --global yarn
+```
+
+### Install Git
+
+Download [Git Installer](https://git-scm.com/download/win) from their Official website.
+
+## Install Dependencies for Mac Users
+
+Download [Nodejs installer](https://nodejs.org/en/download/) from their Official website.
+
+Alternatively:
+
+### Using Homebew
+
+```
+brew install node
+```
+
+### Using MacPorts
+
+```
+sudo port install nodejs17
+```
+
+### Install Yarn
+
+```
+npm install --global yarn
+```
+
+### Install Git
+
+There are several ways to install Git on a Mac. The easiest is probably to install the Xcode Command Line Tools. 
+
+```
+git --version
+```
+
+
+## Install Dependencies for Debian / Ubuntu
+
+```bash
+sudo apt update
+sudo apt install git npm -y
+npm install --global yarn -y
+```
+
+### Install Git
+
+```
+sudo apt install git-all
```

Comment:
git is already installed by the command
```console
sudo apt install git npm -y
```

also `git-all` installs much more than just git. This can be seen https://packages.ubuntu.com/bionic/git-all